snapcraft/extensions/gnome.py

```diff
@@ -148,6 +148,16 @@ def get_root_snippet(self) -> Dict[str, Any]:
             "environment": {
                 "SNAP_DESKTOP_RUNTIME": "$SNAP/gnome-platform",
                 "GTK_USE_PORTAL": "1",
+                "LD_LIBRARY_PATH": append_to_env(
```

Comment:
this should be done in the launcher script

Reply:
Yes, this piece shouldn't be here.

Reply:
Fixed.